web/cobrands/centralbedfordshire/assets.js

```diff
@@ -0,0 +1,62 @@
+(function(){
+
+if (!fixmystreet.maps) {
+    return;
+}
+
+
+var defaults = {
+    http_options: {
+        url: fixmystreet.staging ? "https://tilma.staging.mysociety.org/mapserver/centralbeds" : "https://tilma.mysociety.org/mapserver/centralbeds",
+        params: {
+            SERVICE: "WFS",
+            VERSION: "1.1.0",
+            REQUEST: "GetFeature",
+            SRSNAME: "urn:ogc:def:crs:EPSG::3857"
+        }
+    },
+    max_resolution: 9.554628534317017,
+    geometryName: 'msGeometry',
+    srsName: "EPSG:3857",
+    body: "Central Bedfordshire Council",
+    strategy_class: OpenLayers.Strategy.FixMyStreet
+};
+
+var centralbeds_types = [
+    "CBC",
+    "Fw",
+];
+
+fixmystreet.assets.add(defaults, {
+    http_options: {
+        params: {
+            TYPENAME: "Highways"
+        }
+    },
+    stylemap: fixmystreet.assets.stylemap_invisible,
+    non_interactive: true,
+    always_visible: true,
+    road: true,
+    all_categories: true,
+    usrn: {
+        attribute: 'streetref1',
+        field: 'NSGRef'
+    },
+    actions: {
+        found: function(layer, feature) {
+            fixmystreet.message_controller.road_found(layer, feature, function(feature) {
```

Comment:
Could we filter this at the server side to only return adopted roads and then could use the standard road_found function? Not sure it makes much difference either way I guess.

Reply:
That's a good idea, the less data transferred the better. The current filters and behaviour are subject to change based on client feedback, so when we've got an idea of the final requirements I'll revisit the server-side filters.